spec/templater_spec.rb

```diff
@@ -24,29 +24,31 @@
 
 require 'templater'
 require 'spec_utils'
-require 'fakefs_helper'
 require 'node'
+require 'filesystem'
 
-TEST_TEMPLATE_PATH = File.join(FIXTURES_PATH, 'template.erb')
-REPO_TEST_CONFIG_PATH = File.join(FIXTURES_PATH, 'configs/repo-unit-test.yaml')
-UNSET_PARAMETER_TEMPLATE_PATH = File.join(FIXTURES_PATH, 'unset_parameter_template.erb')
+TEST_TEMPLATE_PATH = '/fixtures/template.erb'
+UNSET_PARAMETER_TEMPLATE_PATH = '/fixtures/unset_parameter_template.erb'
 TEST_HUNTER_PATH = File.join(FIXTURES_PATH, 'cache/hunter.yaml')
 
 
 RSpec.describe Metalware::Templater do
-  def expect_renders(template_parameters, expected, config: Metalware::Config.new)
-    fshelper = FakeFSHelper.new(config)
-    fshelper.clone(FIXTURES_PATH)
-    fshelper.clone_repo(config.repo_path)
+  let :filesystem {
+    FileSystem.setup do |fs|
+      fs.with_fixtures('/', at: '/fixtures')
```

Comment:
I am finding it a tad confusing what you are trying to achieve here. My understanding is that `fs.with_fixtures` clones files from the real `fixtures` dir to the location specified by at. However methods cached by `FileSystem.setup` run after the `FakeFS` is in place and thus you no longer have access to the real files.